lib/rubocop/cop/rspec/factory_girl/create_list.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      module FactoryGirl
+        # Prefer using create_list over n.times { create :obj } calls.
+        #
+        # @example
+        #   # bad
+        #   3.times { create :user }
+        #
+        #   # good
+        #   create_list :user, 3
+        #
+        #   # good
+        #   3.times { |n| create :user, created_at: n.months.ago }
+        class CreateList < Cop
+          MSG = 'Prefer create_list.'.freeze
+
+          def_node_matcher :n_times, '(send (int $_) :times)'
+
+          def_node_matcher :times_block?, <<-PATTERN
+          (block
+            n_times
+            ...
+          )
+          PATTERN
+
+          def_node_matcher :factory_call?, <<-PATTERN
+            (send nil :create (sym $_) $...)
```

Comment:
Some apps I've worked on don't [include `FactoryGirl::Syntax::Methods `](https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#rspec) to enable this shorthand syntax and call `FactoryGirl.create` instead.